@Types/ITechRecordPostPayload.d.ts

```diff
@@ -0,0 +1,11 @@
+import ITechRecord from "./ITechRecord";
+
+export default interface ITechRecordPostPayload {
```

Comment:
This interface is not needed. `ITechRecordWrapper` should take care of everything. Set it as null where you needed to initialise.

Reply:
If I try to set it to null it will say "Type 'null' is not assignable to type 'string'" and I don't want to define systemNumber: string | null; ...so I defined systemNumber: "" as empty string in functions/postTechRecords.ts